staff/sergio-costa/cv/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Sergio Costa</title>
+</head>
+<body>
+    <header>
+        <h1>CV Sergio Costa</h1>
+    </header>
+    <section>
+        <h3>Datos personales</h3>
+        <ul>
+            <li><strong>Direccion:</strong> C/Josep Irla 27</li>
+            <li><strong>Movil:</strong> 676690541</li>
+            <li><strong>Email:</strong> sergio.cs04@gmail.com</li>
```

Comment:
You can use `<a href="mailto: ....>` and `<a href:="tel: ....>` to make others contact you easier.